framework/doc/content/source/outputs/SolutionInvalidityOutput.md

```diff
@@ -40,6 +40,17 @@ If you want to custimized the time interval, you can create a sub-block in `[Out
   []
 []
 ```
+The new solution warning table is as below:
```

Comment:
also mention that no table is output when there are no invalid solution warnings then / timestep_interval is not set

Reply:
I d put this at the bottom under a !alert note